frontend/src/tabs/zoom.pug

```diff
@@ -1,6 +1,7 @@
 #zoom
+  span.large-font Zoom View
   .zoom__title
-    .zoom_title--group.large-font
+    a.zoom_title--group.large-font
```

Comment:
The link is missing from the title.

Reply:
This was something I wanted to confirm too - The original zoom title doesn't have a link, so I left it like that. Should I add the link?

Reply:
Yes I think you should. There's no point leaving a link that directs to nowhere. It also standardises between the two views.